skbio/stats/composition.py

```diff
@@ -806,6 +807,163 @@ def centralize(mat):
     cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
+@experimental(as_of="0.5.7")
+def vlr(x,y, ddof=1, robust=False):
```

Comment:
```suggestion
def vlr(x, y, ddof=1, robust=False):
```

Reply:
While we are at it, we probably want to have 2 different private functions, namely `_robust_vlr` and `_vlr`, where

```python
def _vlr(x, y, ddof=1):
    """ Assumes x and y don't have zeros"""
    return np.var(np.log(x), np.log(y))

def _robust_vlr(x, y, ddof=1):
   x = np.ma.masked_array(x, mask=x == 0)
   y = np.ma.masked_array(y, mask=y == 0)
   x = np.ma.log(x)
   y = np.ma.log(y)
   return np.ma.var(x - y)

def vlr(x, y, ddof=1, robust=False):
    x = closure(x) # checks for negative proportions and normalizes to 1
    y = closure(y)
    if robust:
        return _robust_vlr(x, y, ddof)
    return _vlr(x, y)
```